src/QMCDrivers/DMC/DMC_CUDA.cpp

```diff
@@ -75,6 +78,9 @@ void DMCcuda::checkBounds (std::vector<PosType> &newpos,
 
 bool DMCcuda::run()
 {
+#if USE_NVTX
```

Comment:
I'd rather see these wrapped in macros, They will be easier to strip out if they were unambiguous single line macros.

Reply:
Do you mean #if defined(USE_NVTX), since defined(..) is technically a macro, or writing wrappers for all possible ways to inject NVTX commands? There are a handful of ways to strip #ifdef to next #endif blocks. I would do it with awk but there is also unifdef.

Reply:
Only this place is `#if USE_NVTX`, it needs to be changed to ifdef at least.

Reply:
Yes, that was mistake, definitely need to fix it. Thanks.